### PR TITLE
Add context to why we exist

### PR DESCRIPTION
**Pull request by hackygolucky**
Added some language re: why CommComm was formed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ For more details read the [Community Committee Charter](https://github.com/nodej
 
 ## Why does the Community Committee exist?
 
-The Node.js project has had its fair share of problems regarding diversity, and its fair share of attempts to fix that.
+This Community Committee reflects a formal role and the relevance of the voice of community in the governance of the Node.js project. The formation of this group as a top-level committee alongside the Technical Steering Committee(TSC) is communicated value of the contributors of Node.js that roles other than those of code contributor help foster a healthy, sustainable open source community.
```
Comment (oe) [inline README.md]: the second sentence in this is kind of hard to read, and uses complex language. is there a way to make it less confusing for people who don't know what's going on?

**Comment by oe** [comment]
Thanks a lot for this additional context, Tracy! I appreciate your insight on this and I'll list it as a blocker on the related PR. Do you mind if I remove your commit on the other branch?

**Comment by hackygolucky** [comment]
@oe oh gosh yes. I thought I hadn't made that commit. Sorry! Will also rework this language a little.

File: README.md [hunk 2]
```diff
@@ -6,9 +6,9 @@ For more details read the [Community Committee Charter](https://github.com/nodej
 
 ## Why does the Community Committee exist?
 
-The Node.js project has had its fair share of problems regarding diversity, and its fair share of attempts to fix that.
+The Community Committee reflects a formal role and the relevance of the voice of community in the governance of the Node.js project. The formation of this group as a committee alongside the Technical Steering Committee(TSC) demonstrates that community-focused contributions are valued by the contributors of Node.js–that roles other than those of code contributor help foster a healthy, sustainable open source community.
```
Comment (mhdawson) [inline README.md]: I might suggest changing "-that" with ", and that"

**Comment by chowdhurian** [comment]
LGTM.

**Comment by chowdhurian** [comment]
Is adding a history.md, as mentioned in the comments of #274, within the context of this PR?

It seemed like a suggestion, and there hasn't yet been a discussion around that part specifically.

**Comment by oe** [comment]
@chowdhurian I wouldn't put it within the scope of this PR or its parent, but it's definitely something we could consider.

**Pull request merged**